spec/requests/data_api/tad_exports_spec.rb

```diff
@@ -3,20 +3,92 @@
 RSpec.describe 'GET /data-api/tad-data-exports', type: :request, sidekiq: true do
   include DataAPISpecHelper
 
+  let!(:create_data_export) do
```

Comment:
Better to use a `before` block if we don't need the return values of the `let` blocks?

Reply:
Thanks, I've added one and taken out some logic